tests/validation/validate.cpp

```diff
@@ -54,8 +53,9 @@ int main(int argc, char **argv) {
             }
         }
 
-        rv = RUN_ALL_TESTS();
+        if (RUN_ALL_TESTS());   // Wrap with empty if to silence output not used warning
```

Comment:
if you just want to ignore the return value of `RUN_ALL_TESTS` just run it inline as

```
RUN_ALL_TESTS();
```

But in this case we want to return this value to the shell, i.e.

```
return RUN_ALL_TESTS();
```